src/test/scala/com/microsoft/kusto/spark/KustoSinkTests.scala

```diff
@@ -48,7 +49,7 @@ class KustoSinkTests extends FlatSpec with MockFactory with Matchers with Before
   }
 
   private def getSink: KustoSink =
-    new KustoSink(sqlContext, kustoCluster, kustoDatabase, kustoTable, appId, appKey, appAuthorityId)
+    new KustoSink(sqlContext, KustoTableCoordinates(kustoCluster, kustoDatabase, kustoTable), AadApplicationAuthentication(appId,appKey, appAuthorityId), KustoSparkWriteOptions(writeResultLimit = "none"))
```

Comment:
I saw you added the "none" as constant, but I don't think you use it. Probably no need to define as constant